core/inject.js

```diff
@@ -54,7 +53,7 @@ Blockly.inject = function(container, opt_options) {
         document.querySelector(container);
   }
   // Verify that the container is in document.
-  if (!goog.dom.contains(document, container)) {
+  if (!Blockly.utils.containsNode(document, container)) {
```

Comment:
`container` is an unfortunate variable name for the second parameter, given the container actually tested is `document`.